sdk/service/service.go

```diff
@@ -87,3 +87,8 @@ func (s *Service) Create(srv *service.Service) error {
 
 	return s.db.Save(srv)
 }
+
+// Get returns the service that matches given hash.
+func (s *Service) Get(hash string) (*service.Service, error) {
```

Comment:
personally, hashOrSid is a better name

Reply:
Sid will be replaced by a dedicated system in the future OR will be managed by CLI.
All the API will use only hash as unique identifier of ANY resource.